src/main/docs/guide/cli.adoc

```diff
@@ -6,24 +6,12 @@ $ mn create-app my-rabbitmq-app --features rabbitmq
 
 This will create a project with the minimum necessary configuration for RabbitMQ.
 
-=== RabbitMQ Profile
-
-The Micronaut CLI includes a specialized profile for RabbitMQ based messaging applications. This profile will create a Micronaut app with RabbitMQ support, and _without_ an HTTP server (although you can add one if you desire). The profile also provides a couple commands for generating RabbitMQ consumers and producers.
-
-To create a project using the RabbitMQ profile, use the `profile` flag:
-
-----
-$ mn create-app my-rabbit-service --profile rabbitmq
-----
-
-As you'd expect, you can start the application with `./gradlew run` (for Gradle) or `./mvnw compile exec:exec` (Maven). The application will (with the default config) attempt to connect to RabbitMQ at `http://localhost:5672`, and will continue to run without starting up an HTTP server. All communication to/from the service will take place via RabbitMQ producers and/or consumers.
-
```

Comment:
Do not delete this, instead change `profile` to `feature`. The following works as expected:

```
mn create-app my-rabbit-service --features rabbitmq
```

Reply:
Thanks for you input, but --feature option is already described above, check section 3 in https://micronaut-projects.github.io/micronaut-rabbitmq/latest/guide/

Reply:
Sorry, I missed that part. We need to remove the word `profile` because that's not a thing in Micronaut anymore (it was for Micronaut 1.x).

The right command is:
```
mn create-messaging-app my-rabbit-service --features rabbitmq
```

Using `create-messaging-app` in fact does what the docs explain: create an application without http server.

Reply:
Cool, I wasn't sure if --feature option does create without http server or not. I will leave the last paragraph then.